packages/core/src/generation.ts

```diff
@@ -176,7 +176,8 @@ export async function generateText({
             case ModelProviderName.LLAMACLOUD:
             case ModelProviderName.NANOGPT:
             case ModelProviderName.HYPERBOLIC:
-            case ModelProviderName.TOGETHER: {
+            case ModelProviderName.TOGETHER:
```

Comment:
Please add akash to the other generateX methods as relevant

Reply:
Added to handleProvider function.